app/views/mortgage_calculator/stamp_duties/_bands_table.html.erb

```diff
@@ -1,47 +1,46 @@
-<div class="table-wrapper">
-  <table class="mortgagecalc__table">
-    <thead>
-      <tr>
-        <th class="mortgagecalc__table__price"><%= I18n.t("stamp_duty.table.property_price_header") %></th>
-        <th class="mortgagecalc__table__rate"><%= I18n.t("stamp_duty.table.rate_header") %></th>
-        <th class="mortgagecalc__table__extra"><%= I18n.t("stamp_duty.table.extra_rate_header") %></th>
-      </tr>
-    </thead>
+<table class="mortgagecalc__table stamp-duty__table">
+  <thead>
+    <tr>
+      <th class="mortgagecalc__table__price"><%= I18n.t("stamp_duty.table.property_price_header") %></th>
+      <th class="mortgagecalc__table__rate"><%= I18n.t("stamp_duty.table.rate_header") %></th>
+      <th class="mortgagecalc__table__extra"><%= I18n.t("stamp_duty.table.extra_rate_header") %></th>
+    </tr>
+  </thead>
 
-    <tbody>
-      <tr>
-        <td class="mortgagecalc__table__price">&pound;0 - &pound;125,000</td>
-        <td class="mortgagecalc__table__rate">0%</td>
-        <td class="mortgagecalc__table__extra">3%</td>
-      </tr>
+  <tbody>
+    <tr>
+      <td class="mortgagecalc__table__price">&pound;0 - &pound;125,000</td>
+      <td class="mortgagecalc__table__rate">0%</td>
+      <td class="mortgagecalc__table__extra">3%</td>
+    </tr>
 
-      <tr>
-        <td class="mortgagecalc__table__price">&pound;125,001 - &pound;250,000</td>
-        <td class="mortgagecalc__table__rate">2%</td>
-        <td class="mortgagecalc__table__extra">5%</td>
-      </tr>
+    <tr>
+      <td class="mortgagecalc__table__price">&pound;125,001 - &pound;250,000</td>
+      <td class="mortgagecalc__table__rate">2%</td>
+      <td class="mortgagecalc__table__extra">5%</td>
+    </tr>
 
-      <tr>
-        <td class="mortgagecalc__table__price">&pound;250,001 - &pound;925,000</td>
-        <td class="mortgagecalc__table__rate">5%</td>
-        <td class="mortgagecalc__table__extra">8%</td>
-      </tr>
+    <tr>
+      <td class="mortgagecalc__table__price">&pound;250,001 - &pound;925,000</td>
+      <td class="mortgagecalc__table__rate">5%</td>
+      <td class="mortgagecalc__table__extra">8%</td>
+    </tr>
 
-      <tr>
-        <td class="mortgagecalc__table__price">&pound;925,001 - &pound;1.5 <%= I18n.t("stamp_duty.table.million") %></td>
-        <td class="mortgagecalc__table__rate">10%</td>
-        <td class="mortgagecalc__table__extra">13%</td>
-      </tr>
+    <tr>
+      <td class="mortgagecalc__table__price">&pound;925,001 - &pound;1.5 <%= I18n.t("stamp_duty.table.million") %></td>
+      <td class="mortgagecalc__table__rate">10%</td>
+      <td class="mortgagecalc__table__extra">13%</td>
+    </tr>
+
+    <tr>
+      <td class="mortgagecalc__table__price"><%= I18n.t("stamp_duty.table.over") %> &pound;1.5 <%= I18n.t("stamp_duty.table.million") %></td>
+      <td class="mortgagecalc__table__rate">12%</td>
+      <td class="mortgagecalc__table__extra">15%</td>
+    </tr>
+  </tbody>
+</table>
```

Comment:
That's a great idea Tomas.

All we did here was remove a wrapping div so didn't give it any though.
Will slot this in next time we're in the area.